src/applications/mhv-secure-messaging/tests/e2e/pages/PatientComposePage.js

```diff
@@ -526,25 +526,61 @@ class PatientComposePage {
   };
 
   verifyRecipientsQuantityInGroup = (index, quantity) => {
-    cy.get(Locators.DROPDOWN.RECIPIENTS)
-      .find(`optgroup`)
-      .eq(index)
-      .find('option')
-      .should(`have.length`, quantity);
+    // Try combobox selector first (curated list flow), fallback to dropdown
+    cy.get('body').then($body => {
+      if ($body.find('[data-testid="compose-recipient-combobox"]').length > 0) {
+        cy.get('[data-testid="compose-recipient-combobox"]')
+          .find(`optgroup`)
+          .eq(index)
+          .find('option')
+          .should(`have.length`, quantity);
+      } else {
+        // Deprecated: Remove after combobox is fully rolled out
```

Comment:
I repeated this several times. Since some tests do and don't have the combobox, we adjust based on the presence of the combobox. I marked it as deprecated so that we can remove this logic when combobox is fully rolled out

Reply:
it may be safer to split into separate methods and call on one method when we know we expect specific result (whether it is feature flagged or else), or the other. I like adding a comment too.